From 0662a11ce3b1e83ac15b3a78f77e8210af0466fa Mon Sep 17 00:00:00 2001
From: Mario Alfredo Carrillo Arevalo <mario.alfredo.c.arevalo@intel.com>
Date: Fri, 10 Nov 2017 13:32:17 -0600
Subject: [PATCH] opencv: Update version for building plugins

This commit allows to build the OpenCV
plugins using the 3.3.1 OpenCV version.

Signed-off-by: Mario Alfredo Carrillo Arevalo <mario.alfredo.c.arevalo@intel.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 60afdfb..0d2ccc1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2821,7 +2821,7 @@ AG_GST_CHECK_FEATURE(OPENCV, [opencv plugins], opencv, [
   dnl a new version and the no-backward-compatibility define. (There doesn't
   dnl seem to be a switch to suppress the warnings the cvcompat.h header
   dnl causes.)
-  PKG_CHECK_MODULES(OPENCV, opencv >= 2.3.0 opencv <= 3.3.0 , [
+  PKG_CHECK_MODULES(OPENCV, opencv >= 2.3.0 opencv <= 3.3.1 , [
     AC_PROG_CXX
     AC_LANG([C++])
     OLD_CPPFLAGS=$CPPFLAGS
-- 
2.10.0.372.g6fe1b14

